Bugzilla/Template.pm

```diff
@@ -739,6 +739,22 @@ sub create {
         1
       ],
 
+      renderMarkdown => [
+        sub {
+            my ($context, $bug, $comment, $user) = @_;
+            return sub {
+                my $text = shift;
+                if($comment && $comment->is_markdown) {
```

Comment:
Add && Bugzilla->params->{use_markdown} so we can force the old behavior if needed